packages/in_app_purchase/android/src/main/java/io/flutter/plugins/inapppurchase/InAppPurchasePlugin.java

```diff
@@ -81,7 +85,7 @@ public void onReattachedToActivityForConfigChanges(ActivityPluginBinding binding
 
   @Override
   public void onDetachedFromActivityForConfigChanges() {
-    onDetachedFromActivity();
+    methodCallHandler.setActivity(null);
```

Comment:
Why call this and not `methodCallHandler.onDetachedFromActivity()`?

Reply:
I think it is not necessary to disconnect the billing client when activity detached from config changes (e.g. when the phone is rotated).

Reply:
Makes sense, could you also document that here? Everywhere else we treat these callbacks as equivalent.